python/katana_setup.py

```diff
@@ -202,7 +322,9 @@ def setup(*, source_dir, package_name, doc_package_name, **kwargs):
         package_data={"": [str(f) for f in pxd_files]},
         package_dir={"": str(source_dir)},
         tests_require=["pytest"],
-        setup_requires=setup_requires,
+        # NOTE: Do not use setup_requires. It doesn't work properly for our needs because it does install the packages
```

Comment:
"it does install"?
would either "it doesn't install" or "it installs" be clearer?